pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -382,6 +383,16 @@ func getDiskTargetsForMigration(dom cli.VirDomain, vmi *v1.VirtualMachineInstanc
 	return copyDisks
 }
 
+func loopbackAddress() string {
+	ipString := os.Getenv("KUBERNETES_SERVICE_HOST")
+	if netutils.IsIPv6String(ipString) {
+		return "[::1]"
+	}
+
+	// TODO in case env is "" do we want to return error or to fallback to IPv4 ?
```

Comment:
@AlonaKaplan 
what to do in case of an error ? just fall back to ipv4 ? return err all the way ?
just print and faill back to ipv4 ?

same question on the 2nd function

Reply:
obsolete , see please previous comment